<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>723</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>723</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>640</width>
    <height>723</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Service Students</string>
  </property>
  <widget class="QScrollArea" name="table_area">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>621</width>
     <height>511</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>619</width>
      <height>509</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_progress">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>0</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe Script</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="addBtn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>560</y>
     <width>521</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Import Students from Excel file</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_agency_name">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>611</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600; color:#aa0000;&quot;&gt;Service Students&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteBtn">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>560</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Delete all record</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>620</y>
     <width>261</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(170, 0, 0);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="title">
    <string>Grouping Students</string>
   </property>
   <widget class="QPushButton" name="groupBtn">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>50</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Group</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtNG">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Number of groups</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtNS">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>20</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Number of students</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;OR&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="exportBtn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>630</y>
     <width>351</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Export Service Syndicates to Doc</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>641</width>
     <height>371</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:  rgb(115, 193, 238);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>360</y>
     <width>641</width>
     <height>371</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(41, 22, 111);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_sample">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>620</y>
     <width>621</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>0</y>
      <width>591</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; color:#ffffff;&quot;&gt;Excel file to be uploaded must be formated with 9 columns as shown in the sample below:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>19</y>
      <width>621</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../resources/sample.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_group_inst">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>560</y>
     <width>621</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>621</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt; color:#ffffff;&quot;&gt;Please enter either the number of groups or the number of students per group. But NOT both.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>621</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Double click on any record to edit it. You can change student's syndicate by editing.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <zorder>label_6</zorder>
  <zorder>addBtn</zorder>
  <zorder>deleteBtn</zorder>
  <zorder>groupBox</zorder>
  <zorder>exportBtn</zorder>
  <zorder>label_3</zorder>
  <zorder>table_area</zorder>
  <zorder>label_agency_name</zorder>
  <zorder>label_progress</zorder>
  <zorder>groupBox_sample</zorder>
  <zorder>groupBox_group_inst</zorder>
  <zorder>label_7</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
